monetio/sat/_tempo_l2_no2_mm.py

```diff
@@ -102,13 +103,15 @@ def _open_one_dataset(fname, variable_dict):
             "gas_profile",
             "albedo",
             "temperature_profile",
+            "amf",
             "amf_total",
             "amf_diagnositc_flag",
```

Comment:
Unless it's spelled wrong in the datasets
```suggestion
            "amf_diagnostic_flag",
```